src/example-types/text.js

```diff
@@ -1,6 +1,16 @@
 let _ = require('lodash/fp')
 let unidecode = require('unidecode')
 
+const vRegex = str =>
+  str
+    .replace(/[.?+*|{}[]()]/g, '\\$&')
```

Comment:
We have to be really careful and intentional about where and when we escape things. Unless we have a really compelling reason to do so, we're generally better off not doing it at all.

Reply:
If you use regex, you MUST escape special characters that would screw with the regex. Someone typing one of these characters into the search mean for them to be content, not part of the regex.

Reply:
Yup, wildcard didn't support the operators before so this is safe. Probably worth adding a `regex` operator choice that just passes everything through